web/ivr/ivr.go

```diff
@@ -57,15 +57,15 @@ func newIVRHandler(handler ivrHandlerFn, logType models.ChannelLogType) web.Hand
 			return writeGenericErrorResponse(w, errors.Wrapf(err, "unable to get service for channel: %s", ch.UUID()))
 		}
 
-		// validate this request's signature
-		err = svc.ValidateRequestSignature(r)
```

Comment:
this parses the form which reads the body... before `NewRecorder` has a chance to dump it - we don't see this in the unit tests because they skip Twilio request signature checking